linera-core/src/unit_tests/test_utils.rs

```diff
@@ -874,6 +874,15 @@ where
         }
         assert!(count >= target_count);
     }
+
+    /// Panics if any validator has a nonempty outbox for the given chain.
+    pub async fn check_that_validators_have_empty_outboxes(&self, chain_id: ChainId) {
+        for validator in &self.validator_clients {
+            let guard = validator.client.lock().await;
+            let chain = guard.state.chain_state_view(chain_id).await.unwrap();
+            assert_eq!(chain.outboxes.indices().await.unwrap(), []);
```

Comment:
Could you use `is_empty()` here instead?

Reply:
Right, I can change it back.
On the other hand, I used `assert_eq!` so that it would print the unexpected contents if it fails. Maybe that's a good idea in general, in tests?